packages/core/src/configuration.ts

```diff
@@ -149,7 +149,8 @@ const axisChart: AxisChartOptions = Tools.merge({}, chart, {
 	zoomBar: {
 		top: {
 			enabled: false,
-			type: ZoomBarTypes.GRAPH_VIEW
+			type: ZoomBarTypes.GRAPH_VIEW,
+			refreshRangeAxisLabel: false
```

Comment:
Not sure if this makes sense here. What if in top you set `refreshRangeAxisLabel` to true and in bottom you set it to false? Seems like this would be a `zoomBar` level config. And it should probably be called `updateRangeAxis` or `realtimeRangeAxis`

Reply:
@theiliad
Thanks for the prompt review.
I think your comment makes sense.
The option name is changed to `updateRangeAxis` and moved to `ZoomBarsOptions` in 
https://github.com/carbon-design-system/carbon-charts/pull/766/commits/b9abd4c2abf9c738e059a342cc940cb96f71abde

Reply:
@theiliad 
One more thought, how about moving this configuration to`AxisOptions` ?

Reply:
well that depends. It seems like this is only relevant when you have a zoombar, isn't that the case?

Reply:
Yes, so far this option is required only zoom bar is enabled.